conductor_api/src/conductor/base.rs

```diff
@@ -657,7 +658,7 @@ impl Conductor {
 
     pub fn save_dna(&self, dna: &Dna) -> Result<PathBuf, HolochainError> {
         let mut file_path = self.dna_dir_path().join(dna.address().to_string());
```

Comment:
```suggestion
        let file_path = self.dna_dir_path().join(dna.address().to_string())
            .with_file_name(DNA_EXTENSION);
```

Reply:
Will that really work?  I think not because `set_extension` will add the "." between the file name and the path, but `with_file_name` I don't think will.

Reply:
🙈 Oh, sorry,... I meant:
```suggestion
        let mut file_path = self.dna_dir_path().join(dna.address().to_string())
            .with_extension(DNA_EXTENSION);
```